keyboards/bat43/keymaps/default/keymap.c

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2020 yfuku
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _LOWER,
+    _RAISE
+};
+
+#define L_SPC LT(_LOWER, KC_SPC)
+#define R_ENT LT(_RAISE, KC_ENT)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(
+        KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,             KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_MINS,
+        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_BSPC, KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_RSFT,
+        KC_LCTL, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,             KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RCTL,
+                                   KC_LGUI, KC_LANG2,L_SPC,            R_ENT,   KC_LANG1,KC_RALT,
+                                            KC_A,    KC_B,             KC_C,    KC_D,    KC_E
```

Comment:
I don't actually see where these keys are on the board. Please advise.

Reply:
![7E6F7267-EFDD-4756-A66A-6DC37D6CF3D9_1_105_c](https://user-images.githubusercontent.com/30647434/78419407-6cb64f00-7680-11ea-8a5c-55dac0738200.jpeg)
Apart from the normal switches, there are five sub switches.
I have attached a photo.

Reply:
Ah, I didn't see them on the photos on your Booth page, and I glossed over the 6 tactile switches in the BOM.

That is neat!